types/underscore/index.d.ts

```diff
@@ -5721,7 +5721,7 @@ declare module _ {
         * Wrapped type `object`.
         * @see _.invert
         **/
-        invert(): _Chain<string, Dictionary<string>>;
+        invert(): _Chain<any, Dictionary<any>>;
```

Comment:
why did you change that? according to docs the original deceleration is good https://underscorejs.org/#invert